src/components/App.js

```diff
@@ -76,6 +76,22 @@ class App extends React.Component {
     });
   };
 
+  displayAllOfType = (type) => {
+    if (type === "Venues") {
+      this.setState({
+        searchResults: searchBars('', this.state.bars)
+      });
+    } else if (type === "Shows") {
+      this.setState({
+        searchResults: searchShows('', this.state.bars)
+      });
+    } else if (type === "Queens") {
+      this.setState({
+        searchResults: searchQueens('', this.state.queens)
```

Comment:
My only suggestion for cleaning this up would be a switch statement, but I believe it would be about the same length of code?? So it might be perfectly fine like this.

On the other hand... if we can change searchBars to searchVenues, then we could make it dynamic and have it interpolate `search{type}(....)`

Reply:
I was thinking about that but didn't know if interpolation could be used that way. Let me give that a try real quick before merging